### ISSUE TITLE
Unable to compile on mac

### ISSUE THREAD

**Issue by sa6k0o0** [jfisteus/html2xhtml#14]
make
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-recursive
Making all in src
gcc -DHAVE_CONFIG_H -I. -I..    -std=c99 -g -O2 -MT html.o -MD -MP -MF .deps/html.Tpo -c -o html.o html.c
html.l:137:19: error: expected '}'
#line 138 "html.l"
                  ^
html.c:1170:2: note: to match this '{'
        { /* beginning of action switch */
        ^
html.l:137:19: error: expected '}'
#line 138 "html.l"
                  ^
html.c:1124:3: note: to match this '{'
                {
                ^
html.l:137:19: error: expected '}'
#line 138 "html.l"
                  ^
html.c:1086:1: note: to match this '{'
{
^
3 errors generated.
make[2]: *** [html.o] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

**Comment by gmckenzi** [comment]
I see the issue was closed, but was there a solution?

**Comment by jfisteus** [comment]
The same user that reported the issue closed it almost immediately without further explanation. I assumed therefore that no code changes were needed. Are you affected by the same problem?

**Comment by gmckenzi** [comment]
Yes, unfortunately, the same error. I can see that html.c does end without a closing brace.

**Comment by sa6k0o0** [comment]
Well, I solved the error. The issue is that mac users normally have this problem with any gcc program if they have certain flags set on or off. I forgot what flags exactly need to be set, but they are different from the default for MacOS , and after I compiled it, I reverted the flags the way mac has it out of the box.

**Comment by jfisteus** [comment]
Thanks a lot for the information, @sa6k0o0. Do you know of any web page or document that explains how to do that? It would be quite useful to @gmckenzi and other users facing the same  issue. I'm not familiar with Mac and cannot help with this one.

**Comment by sa6k0o0** [comment]
I know I had to install some packages also.
This is the complete list of what I have installed :
autoconf
gettext
libtool
openssl
automake
icu4c		

I am not sure which of those are really needed to compile this project. And the most tricky part was with gettext:

This is what I ended up doing 
brew link gettext --force
and once I was done I just did 
brew unlink gettext

I got the idea from https://superuser.com/questions/747324/brew-install-gettext-should-i-force-link-it

I used brew to install all of those things.
Hope this helps
